### ISSUE TITLE
Test and Update Microsoft.Online.SharePoint.PowerShell (v16.0.24614.12000)

### ISSUE THREAD

**Issue by schrolla** [cisagov/ScubaGear#999]
# 💡 Summary #

Dependent module Microsoft.Online.SharePoint.PowerShell has released a new version v16.0.24614.12000 with a number of bug fixes.  Test the new version for compatibility with ScubaGear and update compatible versions based on the results.

## Motivation and context ##

The MS SharePoint SDK is used for interacting with SharePoint Online.  The previous version contained a bug that made it incompatible with ScubaGear.  This new version is supposed to resolve that bug.  Updating the versions supported by ScubaGear will make sure it has the latest available features and capabilities for querying M365 services.

## Implementation notes ##

Please provide details for implementation, such as:

- Remove existing versions of Microsoft.Online.SharePoint.PowerShell module from the test system
- Update RequiredVersions.ps1 max version to v16.0.24614.12000 (or latest)
- Run ScubaGear functional tests against multiple tenants and products and validate results (test with user auth, not SP)
- Determine if new version is compatible with PS7 or supports service principals
- If new version works, issue PR to update max version in RequiredVersion.ps1

## Acceptance criteria ##

How do we know when this work is done?

- [ ] Compatibility of latest Microsoft.Online.SharePoint.PowerShell module with ScubaGear has been determined
- [ ] If compatible, PR with updated max version has been merged into ScubaGear
- [ ] If new version supports PS7, note in #97  and unblock
- [ ] If new version supports SP auth, create issue to explore continued need for PnP PowerShell
